frontend/src/app/projects/project-detail/model-overview/model-overview.component.html

```diff
@@ -122,59 +122,37 @@ <h2 class="text-xl font-medium">Models</h2>
           }
         </div>
         <div class="m-2.5">
-          @if (userService.user?.role === "administrator") {
-            <a
-              mat-mini-fab
-              color="primary"
-              matTooltip="Model restrictions"
-              class="!m-1.5"
-              [routerLink]="['model', model.slug, 'restrictions']"
-            >
-              <mat-icon>key</mat-icon>
-            </a>
-          }
-          @if (projectUserService.verifyRole("manager")) {
-            <a
-              mat-mini-fab
-              color="primary"
-              matTooltip="Configure model"
-              class="!m-1.5"
-              [routerLink]="['model', model.slug, 'metadata']"
-            >
-              <mat-icon>settings</mat-icon>
-            </a>
+          @if (
+            !project?.is_archived &&
+            project?.type !== "training" &&
+            model.t4c_models &&
+            projectUserService.verifyPermission("write")
```

Comment:
```suggestion
            projectUserService.verifyPermission("write") &&
            model.version
```

If the model version is not set, we can't request a session. Fixed it already.